docker-compose.yml

```diff
@@ -301,3 +303,8 @@ services:
       DISABLE_UNOCONV: '1'
     ports:
       - 4006:4000
+
+  clamav-server:
+    image: mkodockx/docker-clamav
+    ports:
+      - 3310:3310
```

Comment:
this is going to need the equivalent helm chart / k8s setup I guess?

Reply:
Thank you for the reminder - already done and pushed.